### PR TITLE
Doctrine ORM 2.5.0-alpha2 release blogpost

### PR DESCRIPTION
**Pull request by Ocramius**

### CODE REVIEW

**Comment by Ocramius** [comment]
Still need to tag: review this first plox.

/cc @beberlei @asm89, @deeky666

File: site/2015/01/25/orm-2-5-0-alpha-2.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+You can find the current state of the 2.5.0 changes overview in
+`the upgrade notes <http://docs.doctrine-project.org/en/latest/changelog/migration_2_5.html>`_.
+
+Release RoadMap
+~~~~~~~~~~~~~~~
+
+We expect to release following versions of the ORM in the next days:
+
+ - ``2.5.0-beta1`` on ``2014-01-31``
```
Comment (Ocramius) [inline site/2015/01/25/orm-2-5-0-alpha-2.rst]: I will change these to mondays.

**Comment by deeky666** [comment]
:+1:

**Comment by asm89** [comment]
:+1:

**Pull request merged**